src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -373,16 +359,13 @@ private void addListElements(Document doc, Element parentElement, Collection<?>
             if (entry instanceof Struct) {
                 final Element entryElement = addNode(doc, parentElement, elementName);
                 processStruct(doc, entryElement, (Struct) entry);
-            }
-            else if (entry instanceof Collection) {
+            } else if (entry instanceof Collection) {
                 final Element entryElement = addNode(doc, parentElement, elementName);
                 addListElements(doc, entryElement, (Collection<?>) entry, "entry");
-            }
-            else if (entry instanceof Map) {
+            } else if (entry instanceof Map) {
```

Comment:
```suggestion
            } 
            else if (entry instanceof Map) {
```

Unrelated to this specific PR